File: Assignment1/addition.cpp
/**********************7.write a program to calculate addition of two number**************************/
#include<iostream>
using namespace std;
class Addition
{
	public:
	int num1,num2,add;
	void getdata()
	{
	cout<<"Enter First number:"<<endl;	
	cin>>num1;
	cout<<"Enter second number"<<endl;	
	cin>>num2;
	}
	void cal_addtion()      //fuction addting two number
	{
	add=num1+num2;	     
	}	
	void display()
	{
		cout<<"Addition of"<<num1<<"and"<<num2<<"is:"<<add<<endl;
	}
};
int main()
{
	Addition a1;
	a1.getdata();
	a1.cal_addtion();
	a1.display();
}

File: JAVA/Assignment1/Addition.java
/**********7. WAP to calculate the addition of 2 numbers.**********/
import java.util.*;
class Addition
{
int num1,num2;
void getdata()
{
Scanner sc=new Scanner(System.in); // help to take i/p from user
System.out.println("Enter number");
num1=sc.nextInt();
System.out.println("Enter number");
num2=sc.nextInt();
}
void addition()
{
num1=num1+num2;
}
void display()
{
System.out.println("-----------------");
System.out.println("Addition:"+num1);
System.out.println("-----------------");
}
public static void main(String[] args)
{
Addition a1=new Addition();
a1.getdata();
a1.addition();
a1.display();
}
}